src/main/java/net/snowflake/client/core/SFTrustManager.java

```diff
@@ -656,6 +658,11 @@ private static boolean checkOCSPResponseValidityErrorParameter() {
     return Boolean.TRUE.toString().equalsIgnoreCase(injectValidityError);
   }
 
+  static void resetOcspConfiguration() {
```

Comment:
why do we need this?

Reply:
It's not necessary here - I moved it to the test code